Admin/Admin.php

```diff
@@ -1620,20 +1653,6 @@ public function setFormGroups(array $formGroups)
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function removeFieldFromFormGroup($key)
```

Comment:
> Methods can be called from an extension, so this is a big BC break. We need to keep them

Agree with @rande. As far as the method is public, it could be call anywhere and cause BC break if it is removed.

If this method is really not needed, please keep it with a deprecation mark (`@deprecated` + `trigger_error`) and make another PR to removed it on next major release.